tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1898,4 +1898,134 @@ public function provideTablesForTruncation()
             ['create table t(id integer primary key autoincrement)', 'T', 't'],
         ];
     }
+
+    public function testForeignKeyReferenceCorrectAfterRenameColumn()
+    {
+        $refTable = new \Phinx\Db\Table('ref_table', [], $this->adapter);
+        $refTable->addColumn('columnToRename', 'string')->save();
+
+        $table = new \Phinx\Db\Table('table', [], $this->adapter);
+        $table->addColumn('ref_table_id', 'integer');
```

Comment:
I suggest to move `ref_table_id` string into variable, to avoid copy-pasta and make it more DRY.

Reply:
Fixed:
- all column names used twice or more was moved to variables
- all table names now received by $table->getName()